libs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@audius/libs",
-  "version": "0.11.76",
+  "version": "0.11.77",
```

Comment:
i think @vicky-g also bumped libs in her pr. you two should figure out who bumps it. or we should not do libs bumps in pr's and make them separate pr's

Reply:
un-bumping libs, and making a separate pr to be merged after @vicky-g's change. After everything is in master, we can npm publish.